acceptance: TestDockerCLI/test_copy failed

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9365405?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9365405?buildTab=artifacts#/) on release-23.1 @ [fde1d54b549c7ebbc5139a5c63cb27256566af5b](https://github.com/cockroachdb/cockroach/commits/fde1d54b549c7ebbc5139a5c63cb27256566af5b):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI3052886711
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI3052886711
--- FAIL: TestDockerCLI (865.52s)
=== RUN   TestDockerCLI/test_demo_global.tcl
=== CONT  TestDockerCLI/test_demo_global.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_global.tcl (93.37s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
- #61834 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-master]
</p>
</details>
/cc @cockroachdb/sql-sessions
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-26368

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9440191?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9440191?buildTab=artifacts#/) on release-23.1 @ [e847e4358c004d4a8977ae26749768ad1d417fba](https://github.com/cockroachdb/cockroach/commits/e847e4358c004d4a8977ae26749768ad1d417fba):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2657672395
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2657672395
--- FAIL: TestDockerCLI (1023.35s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (40.47s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9938367?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9938367?buildTab=artifacts#/) on release-23.1 @ [5c79003c3ae57376ccf316877262171209f671c3](https://github.com/cockroachdb/cockroach/commits/5c79003c3ae57376ccf316877262171209f671c3):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2789826538
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2789826538
--- FAIL: TestDockerCLI (973.78s)
=== RUN   TestDockerCLI/test_log_config_msg.tcl
=== CONT  TestDockerCLI/test_log_config_msg.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_log_config_msg.tcl (29.52s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101712 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-23.1.0]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10163829?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10163829?buildTab=artifacts#/) on release-23.1 @ [e7f09b3bb7014cf0e238abc53db04501a31de689](https://github.com/cockroachdb/cockroach/commits/e7f09b3bb7014cf0e238abc53db04501a31de689):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1006772596
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1006772596
--- FAIL: TestDockerCLI (911.67s)
=== RUN   TestDockerCLI/test_notice.tcl
=== CONT  TestDockerCLI/test_notice.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_notice.tcl (34.72s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101712 acceptance: TestDockerCLI failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10427680?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10427680?buildTab=artifacts#/) on release-23.1 @ [1a5dc2aad16ca4185afcda75d923804324dcfeb9](https://github.com/cockroachdb/cockroach/commits/1a5dc2aad16ca4185afcda75d923804324dcfeb9):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1634048459
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1634048459
--- FAIL: TestDockerCLI (995.25s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (37.43s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101712 acceptance: TestDockerCLI failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10657120?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10657120?buildTab=artifacts#/) on release-23.1 @ [dbf6ce8438742a73b9d17391c1ef868a4e7a9569](https://github.com/cockroachdb/cockroach/commits/dbf6ce8438742a73b9d17391c1ef868a4e7a9569):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2713832464
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2713832464
--- FAIL: TestDockerCLI (924.93s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (37.96s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101712 acceptance: TestDockerCLI failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10719893?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10719893?buildTab=artifacts#/) on release-23.1 @ [98790caeec6d8db2c3dd69f787970075d3085efd](https://github.com/cockroachdb/cockroach/commits/98790caeec6d8db2c3dd69f787970075d3085efd):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI490620995
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI490620995
--- FAIL: TestDockerCLI (846.81s)
=== RUN   TestDockerCLI/test_copy.tcl
=== CONT  TestDockerCLI/test_copy.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_copy.tcl (35.76s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101712 acceptance: TestDockerCLI failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/10741522?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/10741522?buildTab=artifacts#/) on release-23.1 @ [46b5a2cdc03a1bcbf59aab077cd062a8db62cc1b](https://github.com/cockroachdb/cockroach/commits/46b5a2cdc03a1bcbf59aab077cd062a8db62cc1b):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI2920872126
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI2920872126
--- FAIL: TestDockerCLI (862.65s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (36.37s)
```
<p>Parameters: <code>GOFLAGS=-json</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101712 acceptance: TestDockerCLI failed [C-test-failure O-robot todo-deprecate.branch-release-23.1.0]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10746644?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10746644?buildTab=artifacts#/) on release-23.1 @ [ac605cf689c2a3786aa4c942449c651c2cfdf5e3](https://github.com/cockroachdb/cockroach/commits/ac605cf689c2a3786aa4c942449c651c2cfdf5e3):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1334246070
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1334246070
--- FAIL: TestDockerCLI (811.81s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (38.79s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101712 acceptance: TestDockerCLI failed [C-test-failure O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot T-sql-foundations branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/10785894?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/10785894?buildTab=artifacts#/) on release-23.1 @ [f1108313b2b51ffab176b6b230bcc15b0259f170](https://github.com/cockroachdb/cockroach/commits/f1108313b2b51ffab176b6b230bcc15b0259f170):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI3068374359
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI3068374359
--- FAIL: TestDockerCLI (869.87s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (38.59s)
```
<p>Parameters: <code>GOFLAGS=-json</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101712 acceptance: TestDockerCLI failed [C-test-failure O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot T-sql-foundations branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10822264?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10822264?buildTab=artifacts#/) on release-23.1 @ [68b7da56e3f2210b40f9f9390548ec272b7f22b7](https://github.com/cockroachdb/cockroach/commits/68b7da56e3f2210b40f9f9390548ec272b7f22b7):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI3904664099
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI3904664099
--- FAIL: TestDockerCLI (910.39s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (37.57s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/10849515?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/10849515?buildTab=artifacts#/) on release-23.1 @ [f62527e83e458f5f4521497e7bfd5c60b70f1e31](https://github.com/cockroachdb/cockroach/commits/f62527e83e458f5f4521497e7bfd5c60b70f1e31):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI3774911506
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
--- FAIL: TestDockerCLI (808.94s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (39.76s)
```
<p>Parameters: <code>GOFLAGS=-json</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10851686?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10851686?buildTab=artifacts#/) on release-23.1 @ [a7d63b0fcba4a8874b171ee58c1c951575659057](https://github.com/cockroachdb/cockroach/commits/a7d63b0fcba4a8874b171ee58c1c951575659057):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI539226825
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI539226825
--- FAIL: TestDockerCLI (810.60s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (38.85s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10941117?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10941117?buildTab=artifacts#/) on release-23.1 @ [5c96635393a0fa869fc478225756c21ca994e079](https://github.com/cockroachdb/cockroach/commits/5c96635393a0fa869fc478225756c21ca994e079):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI3112956410
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI3112956410
--- FAIL: TestDockerCLI (981.91s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (37.73s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

@rafiss looking for some guidance here.

I'm trying to reproduce this on 697747b905573b3f05afe95e49962baf5f4d5a55 on the gceworker with
```
bazel test //pkg/acceptance:acceptance_test --test_filter=TestDockerCLI/test_demo_node_cmds --test_arg -test.v --test_arg -show-logs --test_arg=-l=/home/yuzefovich/go/src/github.com/cockroachdb/cockroach/artifacts/acceptance --test_env=TZ=America/New_York --test_arg=-b=/home/yuzefovich/go/src/github.com/cockroachdb/cockroach/artifacts/cockroach-short --runs_per_test 1 --sandbox_writable_path=/home/yuzefovich/go/src/github.com/cockroachdb/cockroach/artifacts/acceptance
```
but I'm getting:
```
expect: does " " (spawn_id exp3) match glob pattern "invalid syntax: \\demo node. Try \\? for help."? no^M
root@127.0.0.1:26257/defaultdb> \demo node^M
^M
expect: does " \demo node\r\n" (spawn_id exp3) match glob pattern "invalid syntax: \\demo node. Try \\? for help."? no^M
ERROR: invalid syntax: \demo node^M
HINT: Try \? for help.^M
^M
expect: does " \demo node\r\nERROR: invalid syntax: \demo node\r\nHINT: Try \? for help.\r\n" (spawn_id exp3) match glob pattern "invalid syntax: \\demo node. Try \\? for help."? no^M
^M
expect: does " \demo node\r\nERROR: invalid syntax: \demo node\r\nHINT: Try \? for help.\r\nroot@127.0.0.1:26257/defaultdb> " (spawn_id exp3) match glob pattern "invalid syntax: \\demo node. Try \\? for help."? no^M
expect: timed out^M
```
If I apply the following diff:
```
diff --git a/pkg/cli/interactive_tests/test_demo_node_cmds.tcl b/pkg/cli/interactive_tests/test_demo_node_cmds.tcl
index 88e8a7a0cc8..0e1697990c3 100644
--- a/pkg/cli/interactive_tests/test_demo_node_cmds.tcl
+++ b/pkg/cli/interactive_tests/test_demo_node_cmds.tcl
@@ -11,7 +11,7 @@ eexpect "defaultdb>"
 
 # Wrong number of args
 send "\\demo node\r"
-eexpect "invalid syntax: \\\\demo node. Try \\\\? for help."
+eexpect "invalid syntax"
 eexpect "defaultdb>"
 
 # Cannot shutdown node 1
```
the test passes. Am I invoking it incorrectly somehow?

Also, when I try to "stress" this by running with `--runs_per_test 10`, I get:
```
    cli_test.go:91: Error response from daemon: Conflict. The container name "/docker-cli_test" is already in use by container "c7c0986421ba726414d7057203a94a22a5cca4916a97a35c7dd7650c217e667d". You have to remove (or rename) that container to be able to reuse that name.
=== CONT  TestDockerCLI
```
but I think it worked on master. Is some backport missing to enable that?

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/11128140?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/11128140?buildTab=artifacts#/) on release-23.1 @ [3e83c0f4e4a110efdd47ecdfd4cb793a3def3c0d](https://github.com/cockroachdb/cockroach/commits/3e83c0f4e4a110efdd47ecdfd4cb793a3def3c0d):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1595642664
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1595642664
--- FAIL: TestDockerCLI (901.40s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (12.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/11192770?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/11192770?buildTab=artifacts#/) on release-23.1 @ [433312f2e8f7963e1eaee181fb46650aaf86cdac](https://github.com/cockroachdb/cockroach/commits/433312f2e8f7963e1eaee181fb46650aaf86cdac):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI58077387
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI58077387
--- FAIL: TestDockerCLI (952.26s)
=== RUN   TestDockerCLI/test_copy.tcl
=== CONT  TestDockerCLI/test_copy.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_copy.tcl (35.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cli/interactive_tests/common.tcl
+++ b/pkg/cli/interactive_tests/common.tcl
@@ -17,7 +17,7 @@ system "rm -f $histfile"
 
 # Everything in this test should be fast. Don't be tolerant for long
 # waits.
-set timeout 30
+set timeout 45
 
 # When run via Docker the enclosing terminal has 0 columns and 0 rows,
 # and this confuses readline. Ensure sane defaults here.
@@ -63,6 +63,7 @@ proc handle_timeout {text} {
     exit 1
 }
 proc eexpect {text} {
+    system "echo; echo \$(date '+.%y%m%d %H:%M:%S.%N') START EXPECT TEST | tee -a logs/expect-cmd.log"
     expect {
 	$text {}
 	timeout { handle_timeout $text }

